tests/php/ORM/DBCompositeTest.php

```diff
@@ -111,4 +113,35 @@ public function testInheritedTables()
         $this->assertEquals('DBCompositeTest_SubclassedDBFieldObject', $object2->dbObject('OtherMoney')->getTable());
         $this->assertEquals('DBCompositeTest_SubclassedDBFieldObject', $object2->dbObject('OverriddenMoney')->getTable());
     }
+
+    /* Uncomment this for SS5
+    public function testSetFieldDynamicPropertyException()
+    {
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage(implode(' ', [
+            'Field abc does not exist.',
+            'If this was accessed via a dynamic property then call setDynamicData() instead.'
+        ]));
+        $object = new DBCompositeTest\TestObject();
+        $object->MyMoney->abc = 'def';
+    }
+    */
```

Comment:
```suggestion
```
Please just delete this instead of commenting it out.
We'll need a PR to include this for CMS 5 whether it's included here as a comment or not, so having this comment doesn't do anything as far as getting this code included in CMS 5 - and it pollutes the CMS 4 codebase in the meantime.